api/suggestion/models/suggestion.js

```diff
@@ -1,24 +1,48 @@
 "use strict";
 const { createLifecycleHooks } = require("../../../utils/algolia");
 
-/**
- * Read the documentation (https://strapi.io/documentation/developer-docs/latest/development/backend-customization.html#lifecycle-hooks)
- * to customize this model
- */
 const contentType = "suggestion";
 
+const notifyAboutSuggestion = async (suggestion) => {
+  const settings = await getSettings();
+
+  const recipients = settings.suggestion_notification_recipients?.split(",");
+
+  await Promise.all(
+    recipients.map(
+      async (recipient) =>
+        await sendSuggestionNotification(recipient, suggestion)
+    )
+  );
+};
+
+const getSettings = async () => {
+  return await strapi.services.settings.find();
+};
+
+const sendSuggestionNotification = async (recipient, suggestion) => {
+  console.log("Sending new suggestion notification to:", recipient);
+
+  const emailConfig = {
+    to: recipient,
```

Comment:
I *think* this field can be a list of the recipients but I couldn't test it.

Reply:
It can be, but then all recipients see each other. I think it's better to send separate emails.